controllers/dscinitialization/dscinitialization_test.go

```diff
@@ -48,7 +48,8 @@ var _ = Describe("DataScienceCluster initialization", func() {
 			Expect(foundMonitoringNamespace.Name).Should(Equal(monitoringNamespace))
 		})
 
-		It("Should create default network policy", func() {
+		// Currently commented out in the DSCI reconcile - setting test to Pending
+		PIt("Should create default network policy", func() {
```

Comment:
P ?

Reply:
Yes, otherwise the test will fail as the code for creating NetworkPolicy is commented out. `PIt` - marks test case as Pending in the execution report. You can also `XIt` - exclude it, and `FIt` - focus test suite to run only tests with `F`

Reply:
I was hoping that the comment one line above makes it clear :)

Reply:
ah, nice ! new things learnt every day